cherrelle-morrison/08/exercise.test.js

```diff
@@ -0,0 +1,80 @@
+describe('08', () => {
+    /**
+     * You are the manager of a new Vehicle company
+     *
+     *  And you need to create different kind of cars to list in your inventory. 
+     *
+     *  1. Create a Vehicle parent
+     *      - wheels as a property
+     *      - bodyType as a property
+     *
+     *  2. Create different kind of vehicles from the parent
+     *      - each vehicle should have a color
+     *      - each vehicle should have a method called "whatIsMyName" that return the bodyType of the vehicle
+     *      - Example: "I am a sedan"
+     */
+
+    test('Create a Vehicle class, and create different type of vehicles that extends', () => {
+        // Your code here
+        class Vehicle{
+            constructor(wheels, bodyType){
+            this.wheels = wheels;
+            this.bodyType = bodyType;
```

Comment:
Since classes are used to reduce code,  you were expected to define the `whatIsMyName` method on the "Vehicle" class. So that way you wouldn't have to add it to all of the child classes.